### ISSUE TITLE
ohos中json字符串末尾分号丢失

### ISSUE THREAD

**Issue by chenqinggang001** [Tencent/MMKV#1294]
复现demo
```
onCreate(want: Want, launchParam: AbilityConstant.LaunchParam): void {
    hilog.info(0x0000, 'testTag', '%{public}s', 'Ability onCreate');
    let appCtx = this.context.getApplicationContext();
    let mmkvRootDir = MMKV.initialize(appCtx);
    let mmkv = MMKV.defaultMMKV();

    let str: string = '{"courseId":39575,"name":"通识课测试2","type":4,"status":1,"limit":1819956082000,"needapprove":"0","lastModifyDate":1712821563000,"openCourse":1,"md5":"967cdcbd284197d68efb19ca9b27f06b"}';
    mmkv.encodeString('string', str);
    hilog.info(0x0000, 'testTag', '%{public}s', 'decodeString:' + mmkv.decodeString('string'));

  }
```

输出日志：
05-09 10:55:05.817   43921-43921  A00000/testTag                 pid-43921             I     decodeString:{"courseId":39575,"name":"通识课测试2","type":4,"status":1,"limit":1819956082000,"needapprove":"0","lastModifyDate":1712821563000,"openCourse":1,"md5":"967cdcbd284197d68efb19ca9b27f06b"

**Comment by chenqinggang001** [comment]
刚刚好这个json字符串会遇到这个问题，其他的json字符串正常，很奇怪

**Comment by lingol** [comment]
Noted. I will look at it later.

**Comment by lingol** [comment]
It's fixed in the dev branch.

**Comment by lingol** [comment]
It's fixed in v1.3.6-beta on OHPM.

**Comment by xinle** [comment]
> It's fixed in v1.3.6-beta on OHPM.

为啥现在还不发布到正式版本？

**Comment by fgh345** [comment]
同问 什么时候可以更新

**Comment by lingol** [comment]
It's fixed in [v1.3.6](https://github.com/Tencent/MMKV/releases/tag/v1.3.6).
